src/angular/loader/loader.component.html.ts

```diff
@@ -0,0 +1,13 @@
+export default `
+<div class = "sdc-loader-wrapper" *ngIf="!global">
+    <div class="sdc-loader-background" [ngClass]="{'sdc-hide-loader':active < 1}">
+     <div class="sdc-loader {{ size }}" [ngClass]="{'sdc-hide-loader':active < 1}"></div>
+    </div>
```

Comment:
Instead of using css `z-index: -1` to hide the loader, maybe it would be better to use `*ngIf`.